app/controllers/plan_wizard_controller.rb

```diff
@@ -1,13 +1,18 @@
 class PlanWizardController < ApplicationController
   include Wicked::Wizard
 
-  steps :set, :project, :product, :dispatch
+  steps :set, :project, :product, :data_release_strategy, :dispatch
```

Comment:
This cause the step to appear as "Data_release_strategy" when the step is listed on the Work Plans homepage. Could it be displayed as "Data Release Strategy" instead?